modules/swagger-codegen/src/main/resources/TypeScript-Fetch/api.mustache

```diff
@@ -93,12 +93,25 @@ export const {{classname}}FetchParamCreator = {
 
         let contentTypeHeader: Dictionary<string> = {};
 {{#hasFormParams}}
-        contentTypeHeader = { "Content-Type": "application/x-www-form-urlencoded" };
-        fetchOptions.body = querystring.stringify({
-        {{#formParams}}
-            "{{baseName}}": params["{{paramName}}"],
-        {{/formParams}}
-        });
+        let formData: Dictionary<any> = {};
+        let isMultipartData = false;
+{{#formParams}}
+        {{#isFile}}isMultipartData = true;
+        {{/isFile}}formData["{{baseName}}"] = params["{{paramName}}"];
+{{/formParams}}
+
+        if (isMultipartData) {
```

Comment:
I originally used a very similar implementation to the one you suggested and also mentioned in #3921 but noticed that, since `{{#isFile}}` is scoped to the individual parameter, it didn't work for requests that included a file and regular form parameters. I checked again and, unless I'm looping through the individual `formParams`, `{{#isFile}}` always maps to false.

Is there a Mustache template variable available that tracks if _any_ of the included `FormData` attributes is of type file? If so, that would allow me to simplify the mustache template to essentially match your suggestion and get rid of the seemingly extraneous code.